src/main.js

```diff
@@ -8,6 +8,7 @@ const baseDir = core.getInput('base-directory') || '.';
 const ignoreTestFailures = core.getInput('ignore-test-failures') == 'true';
 const shouldCreateStatusCheck = core.getInput('create-status-check') == 'true';
 const shouldCreatePRComment = core.getInput('create-pr-comment') == 'true';
+const shouldUpdateCommentOnChange = core.getInput('update-pr-comment-on-change') == 'true';
```

Comment:
I do have a strong preference for the naming of the variable, sorry I was out Friday and didn't see your question until today.

```suggestion
const updateCommentIfOneExists = core.getInput('update-comment-if-one-exists') == 'true';
```

Reply:
👍